code/style.css

```diff
@@ -1,3 +1,110 @@
+* {
+  margin: 0;
+  border: 0;
+  padding: 0;
+  font-family: 'Montserrat', sans-serif;
+}
 body {
-  background: #FFECE9;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: #fff;
+}
+.ghlogo {
+  width: 30px;
+  height: 30px;
+  margin-right: 5px;
+}
+h1 {
+  padding: 20px;
+}
+h2 {
+  margin: 10px;
+}
+
+h3 {
+  text-transform: uppercase;
+  width: 100%; 
+  text-align: center; 
+  border-bottom: 1px solid #000; 
+  line-height: 0.1em;
+  margin: 10px 0 20px; 
+  font-size: 20px;
+}
+
+
+h3 span { 
+   background:#fff; 
+   padding:0 10px; 
+}
+
+a {
+  text-decoration: none;
+  font-weight: 600;
+  color: black;
+}
+
+a:hover {
+  color: #7a7a7c;
+}
+h3 a:hover {
+  color: #000;
+  font-size: 22px;
+  box-shadow: 5px 5px #7a7a7c;
+}
+
+.bolded {
+  font-weight: 600;
+}
+.profile {
+  text-align: center;
+}
+
+.profile-image {
+  height: 150px;
+  width: 150px;
+  border-radius: 50%;
+  padding: 30px;
+}
+
+.card {
+  border: 1.5px solid rgb(27, 27, 27);
+  box-shadow: 10px 10px #4d4c4e;
+  padding: 20px;
+  margin: 20px;
+}
+
+.chart-wrapper {
+  display: flex;
+  margin: auto;
+  padding: 20px;
+}
+
+.pointing-right {
+  width: 20px;
+}
+
+footer {
+  font-size: 12px;
+}
+
+@media (min-width: 668px) {
+ .project-card {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+ }
+ .chart-wrapper {
+  width: 50%;
+}
+
+}
```

Comment:
Small things, just inconsistent indentation and spacing here and there throughout the style sheet.